### PR TITLE
BaSP-TG-135: Redux apply on timesheets

### PR DESCRIPTION
**Pull request by Juanoauil**

### CODE REVIEW

**Review by gregorioelias — APPROVED** [review]
When wanting to edit a single field of some timesheet, it requires that all the fields be modified, otherwise it sends an error.

**Review by dayanagm96 — APPROVED** [review]
For some reason I cannot add a new timesheet. Fixing that and Gregorio's comment the entity would be working fine

**Review by marcomigliaro — APPROVED** [review]
Same comment as Grego, everything else works fine

File: src/Components/redux/modules/projects/actions.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  POST_PROJECT_SUCCESS,
+  POST_PROJECT_PENDING,
+  POST_PROJECT_ERROR,
+  PUT_PROJECT_SUCCESS,
+  PUT_PROJECT_PENDING,
+  PUT_PROJECT_ERROR,
+  DELETE_PROJECT_SUCCESS,
+  DELETE_PROJECT_PENDING,
+  DELETE_PROJECT_ERROR
+  // GET_BY_ID_PROJECT_SUCCESS,
```
Comment (cocolobos) [inline src/Components/redux/modules/projects/actions.js:14]: do we need this ?

File: src/Components/redux/modules/projects/reducer.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
+import {
+  GET_PROJECT_SUCCESS,
+  GET_PROJECT_PENDING,
+  GET_PROJECT_ERROR,
+  POST_PROJECT_SUCCESS,
+  POST_PROJECT_PENDING,
+  POST_PROJECT_ERROR,
+  // PUT_PROJECT_SUCCESS,
```
Comment (cocolobos) [inline src/Components/redux/modules/projects/reducer.js:8]: same this

File: src/Components/redux/modules/projects/reducer.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+} from './constants';
+
+const initialState = {
+  list: [],
+  isLoading: false,
+  error: '',
+  selectedItem: {}
+};
+
+// let editProject = [];
```
Comment (cocolobos) [inline src/Components/redux/modules/projects/reducer.js:26]: remove this

File: src/Components/redux/modules/projects/reducer.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        isLoading: true,
+        error: initialState.error
+      };
+    case GET_PROJECT_ERROR:
+      return {
+        ...state,
+        isLoading: false,
+        error: action.payload
+      };
+    // case GET_BY_ID_PROJECT_PENDING:
```
Comment (cocolobos) [inline src/Components/redux/modules/projects/reducer.js:48]: do we need this ?

**Review by JulianDeme — APPROVED** [review]

**Pull request merged**